### DISCUSSION TITLE
Are Country Codes and Names bundled in?

### DISCUSSION THREAD

**Discussion by Axum25** [MohmmedAshraf/blade-flags#3] [Q&A] [answered]
First off I would like to say I like the icons, are the country names and [ISO 3166-1 alpha-2 codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) included in this package?

**Comment by MohmmedAshraf** [comment, marked as answer]
Thanks for your feedback, the package doesn't provide country names, only country flags included.
